Gemfile

```diff
@@ -13,6 +13,10 @@ gem 'sassc', '~> 2.0', '>= 2.0.1'
 gem 'stripe', '~> 1.58' # January 19, 2017 version of the Stripe API https://stripe.com/docs/api
 gem 'webpacker', '~> 5.4.3'
 gem 'good_job', '~> 3.6'
+gem 'net-http'
+gem 'net-smtp'
+gem 'net-imap'
+gem 'uri', '0.10.0'
```

Comment:
Hi @wwahammy, I'm pair programming with @GabriellyAssuncao, and this gems was recommendations from @clarissalimab to solve some troubles that occurs when we tryed to startup and build the application. Is this gems useless? At least, after we add this gems some troubles was solved.

Reply:
Hmmm, I don't know quite why the build failed so I guess it would help if I knew. Could you explain what you're experiencing when you don't change those gems?